benchmarks/helics/PholdFederate.hpp

```diff
@@ -9,37 +9,39 @@ SPDX-License-Identifier: BSD-3-Clause
 #include "helics/application_api/MessageFederate.hpp"
 #include "helics/core/ActionMessage.hpp"
 #include "helics/helics-config.h"
+
 #include <random>
 
 using namespace helics;
 /** class implementing a federate for the PHOLD benchmark*/
 class PholdFederate
 {
   public:
-    helics::Time deltaTime = helics::Time (10, time_units::ns);  // sampling rate
-    helics::Time finalTime = helics::Time (10000, time_units::ns);  // final time
-    int evCount = 0; // number of events handled by this federate
+    helics::Time deltaTime = helics::Time(10, time_units::ns);  // sampling rate
+    helics::Time finalTime = helics::Time(10000, time_units::ns);  // final time
+    int evCount = 0;  // number of events handled by this federate
 
   private:
     std::unique_ptr<helics::MessageFederate> mFed;
     helics::Endpoint* ept;
 
     int index_ = 0;
     int maxIndex_ = 0;
-  
+
     // values from paper "Warp Speed: Executing Time Warp on 1,966,080 Cores"
     // 16 circulating events per LP (also ran with 8 events per LP)
     // 10% remote communications (also ran with 25%, 50%, and 100% remote communications)
     // mean exponential distribution of 0.90
     // optional lookahead of .1 (to prevent arbitrarily small increases in time with conservative simulators)
-    unsigned int initEvCount_ = 16; // starting number of events
-    double localProbability_ = .9; // probability of local events
-    double randTimeMean_ = deltaTime * .9; // mean for the exponential distribution used when picking event times
+    unsigned int initEvCount_ = 16;  // starting number of events
```

Comment:
1 space or 2 before trailing comments?    Any rational one way or another?

Reply:
I don't see a reason for ever having two. One space is used consistently to separate tokens / expressions, so this is what I would do here as well.

Reply:
I will update this